src/eq_fault_geom/geomio/cfm_shp_to_vtk.py

```diff
@@ -0,0 +1,214 @@
+from pathlib2 import Path
+import string
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import LineString
+import numpy as np
+import meshio
+# import pdb
+# pdb.set_trace()
+
+# Epsilon value.
+eps = 5000.0
+
+# Cell type.
+cell_type = 'triangle'
+# cell_type = 'quad'
+
+# Output directory and file suffix.
+output_dir = "../../../data/cfm_shapefile/cfm_vtk"
+vtk_suffix = ".vtk"
+
+# Create output directory if it does not exist.
+output_path = Path(output_dir)
+output_path.mkdir(parents=True, exist_ok=True)
+
+# Which dip and depth values to use.
+dip_use = 'Dip_Best'
+depth_use = 'Depth_Max'
+
+# Dictionary for replacing numbers with letters.
+string_list = string.ascii_uppercase[0:10]
+num_replace_dict = {str(idx + 1):letter for idx, letter in enumerate(string_list)}
+
+# Default elevation for bottom of faults.
+default_elev = -50000.0
+
+# Default length for horizontal faults.
+default_length = 50000.0
+
+# Dictionary of direction vectors.
+pi4 = 0.25*np.pi
+direction_vecs = {"N": np.array([ 0.0,  1.0,  0.0], dtype=np.float64),
+                  "E": np.array([ 1.0,  0.0,  0.0], dtype=np.float64),
+                  "S": np.array([ 0.0, -1.0,  0.0], dtype=np.float64),
+                  "W": np.array([-1.0,  0.0,  0.0], dtype=np.float64),
+                  "NE": np.array([np.cos(pi4),  np.sin(pi4),  0.0], dtype=np.float64),
+                  "SE": np.array([np.cos(-pi4),  np.sin(-pi4),  0.0], dtype=np.float64),
+                  "SW": np.array([np.cos(5.0*pi4),  np.sin(5.0*pi4),  0.0], dtype=np.float64),
+                  "NW": np.array([np.cos(3.0*pi4),  np.sin(3.0*pi4),  0.0], dtype=np.float64)}
+
+
+def calculate_dip_rotation(line: LineString, dip_dir: str):
```

Comment:
Are any of these functions general enough that they should be in their own module(s)? They look like they could be used elsewhere??

Reply:
I have now separated all of the functions out into their own modules.